### PR TITLE
fix: (Core) RTL mode arrows is showing wrong in Panel

### PR DESCRIPTION
**Pull request by olshansky**
#### Please provide a link to the associated issue.
Related https://github.com/SAP/fundamental-ngx/issues/3674 _(with [this PR](https://github.com/SAP/fundamental-styles/pull/1861) would close the issue)_
#### Please provide a brief summary of this pull request.
Arrow-button looks into wrong side and has an extra/redundant icon 
image

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: libs/core/src/lib/panel/panel.component.ts [hunk 1]
```diff
@@ -74,7 +74,10 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit {
     panelContent: PanelContentDirective;
 
     /** @hidden */
-    constructor(private _cdRef: ChangeDetectorRef, private _elementRef: ElementRef) {}
+    _rtl$ = this._rtlService.rtl;
+
+    /** @hidden */
+    constructor(private _cdRef: ChangeDetectorRef, private _elementRef: ElementRef, private _rtlService: RtlService) {}
```
Comment (JKMarkowski) [inline libs/core/src/lib/panel/panel.component.ts:80]: RtlService should have Optional decorator

**Comment by JKMarkowski** [comment]
image
It seems like there is double arrow on rtl panel

File: libs/core/src/lib/panel/panel.component.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
             fd-button
             fdType="transparent"
             class="fd-panel__button"
-            [glyph]="expanded ? 'slim-arrow-down' : 'slim-arrow-right'"
+            [glyph]="expanded ? 'slim-arrow-down' : (_rtl$ | async) ? 'slim-arrow-left' : 'slim-arrow-right'"
```
Comment (salarenko) [inline libs/core/src/lib/panel/panel.component.html:7]: Keep the UI simple, evaluate in the TS

**Review by salarenko — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/panel/panel.component.ts (hunk 1 above)
Comment (olshansky) [reply libs/core/src/lib/panel/panel.component.ts]: @JKMarkowski fixed

File: libs/core/src/lib/panel/panel.component.html (hunk 2 above)
Comment (olshansky) [reply libs/core/src/lib/panel/panel.component.html]: @salarenko fixed

**Comment by olshansky** [comment]
> image
> It seems like there is double arrow on rtl panel

@JKMarkowski this issue fixed in https://github.com/SAP/fundamental-styles/pull/1861

**Comment by JKMarkowski** [comment]
@olshansky could you temporary remove this second arrow, by adding some styles into `panel.component.scss`?

**Review by JKMarkowski — APPROVED** [review]

**Comment by kavya-b** [comment]
Seems like Platform Panel `Expanded Panel` example button has recently broken again. Could you add the fix(just add `label` input binding on `fdp-button` and remove the button label from content projection) in this PR itself?
image

**Review by mikerodonnell89 — APPROVED** [review]

File: libs/core/src/lib/panel/panel.component.ts [hunk 3]
```diff
@@ -105,4 +122,23 @@ export class PanelComponent implements CssClassBuilder, OnChanges, OnInit {
...
+            ? 'slim-arrow-down'
+            : this._rtl
+                ? 'slim-arrow-left'
+                : 'slim-arrow-right'
+    }
+
+    /** @hidden */
+    private _listenRtl(): void {
+        this._subscription.add(
+            this._rtlService.rtl.subscribe(rtl => {
```
Comment (salarenko) [inline libs/core/src/lib/panel/panel.component.ts:138]: You are not checking for `_rtlService` (this is optional dependency).

**Review by salarenko — CHANGES_REQUESTED** [review]

File: libs/core/src/lib/panel/panel.component.ts (hunk 3 above)
Comment (olshansky) [reply libs/core/src/lib/panel/panel.component.ts]: @salarenko thank you, fixed

**Review by salarenko — APPROVED** [review]

**Pull request merged**